### DISCUSSION TITLE
Multisig of Multisigs using multisig_account

### DISCUSSION THREAD

**Discussion by kmchmk** [aptos-labs/aptos-developer-discussions#442] [Questions] [answered]
### Discord user ID

721329822344937482

### Describe your question in detail.

I'm trying to implement a multisig of multisigs using the multisig_account contract in the core framework. Has anyone ever tried this? Is this possible/impossible? Anyone has any example code? Please let me know. Thanks!

### Which operating system are you using?

macOS

### Which SDK or tool are you using? (if any)

TypeScript SDK

**Comment by gregnazario** [comment, marked as answer]
I guess it would be mostly possible, except for the final multisig execution.

Note that, I would generally *not recommend* multisig of multisigs, as it will create a lot of operational burden / you are more likely to lose access to it because of the complexity.

That being said, if you're looking for a multisig of multisigs, I suggest you do a two level:
1. `MultiKey` which is off-chain multisig, these will be your accounts used for submitting approvals to the second level
2. `On chain multisig` which will take in approvals from the `MultiKey`

**Reply by kmchmk** [reply]
Thank you very much. This worked well.
